test/master-playlist-controller.test.js

```diff
@@ -5916,6 +5916,41 @@ QUnit.test('switch playlists if current playlist gets excluded and re-include if
   this.env.log.warn.callCount = 0;
 });
 
+QUnit.test('Playlist is blacklisted indefinitely if number of retries exceeds maxPlaylistRetries', function(assert) {
```

Comment:
We are trying to move from `blacklist` to `exclude` in our wording. and while it is inconsistent for now, we have it on our roadmap to refactor everything. 
```suggestion
QUnit.test('Playlist is excluded indefinitely if number of retries exceeds maxPlaylistRetries', function(assert) {
```